recipes/patchelf/patchelf.inc

```diff
@@ -0,0 +1,12 @@
+SRC_URI = "http://nixos.org/releases/${PN}/${P}/${P}.tar.bz2"
+LICENSE = "GPLv3"
+SUMMARY = "Tool to allow editing of RPATH and interpreter fields in ELF binaries"
+
+inherit autotools c++ c
+
+RECIPE_TYPES = "native canadian-cross"
```

Comment:
I took a brief look at it, and I believe that any patchelf command can be used on any ELF binary.  So it should be changed to sdk instead of canadian-cross.